angularjs-portal-home/src/main/webapp/js/app-config.js

```diff
@@ -25,7 +25,7 @@ define(['angular'], function(angular) {
             },
             'notificationsURL' : '/web/samples/notifications.json',
             'groupURL' : '/portal/api/groups',
-            'kvURL' : '/portal/api/keyvalue/'
+            'kvURL' : '/storage/'
```

Comment:
Ahhh, I just looked.  I didn't realize there was actual mock data there.  I'll close this and we can overlay.